converters/fromZigbee.js

```diff
@@ -5560,6 +5560,84 @@ const converters = {
             return result;
         },
     },
+    wiser_smart_thermostat_client: {
+        cluster: 'hvacThermostat',
+        type: 'read',
+        convert: async (model, msg, publish, options, meta) => {
+            const response = {};
+            if (msg.data[0] == 0xe010) {
```

Comment:
I checked how to implement this but this would get a bit dirty. (since we don't know the dataType). I propose to add these attributes to `cluster.ts` with a `manufacturerCode` instead (like: https://github.com/Koenkk/zigbee-herdsman/commit/3eed20c2861a3a5024b751baaa841e310322f4b1)

Reply:
As mentioned in Koenkk/zigbee-herdsman#363, these devices don't set the MS bit, so they wouldn't be recognised by the handler with `manufacturerCode` set.  The product's behaviour is non-standard (they're older, ZHA 1.2-ish devices, with some internal Schneider cruft) so perhaps its better to keep the brain-dead stuff in the handler rather than messing up the rest of the code.

Another option would be extending the api to allow the datatype to be embedded for "anonymous" attributes, like this:
`endpoint.saveClusterAttributeKeyValue('hvacThermostat', {0xe010: {value: 1, type: 0x30}});` but at first glance the handling for this in typescript is a bit tricky.

Reply:
Totally agree